src/main/java/io/fabric8/jenkins/openshiftsync/BuildConfigToJobMapper.java

```diff
@@ -116,88 +133,103 @@ public static FlowDefinition mapBuildConfigToFlow(BuildConfig bc) throws IOExcep
 	 * @return true if the BuildConfig was changed
 	 */
 	public static boolean updateBuildConfigFromJob(WorkflowJob job, BuildConfig buildConfig) {
```

Comment:
we need additional comments for this method especially as it contains many return statements